x-pack/plugins/uptime/public/pages/page_header.tsx

```diff
@@ -59,6 +63,15 @@ export const PageHeader = React.memo(
             </EuiButtonEmpty>
           </Link>
         </EuiFlexItem>
+        <EuiFlexItem grow={false}>
+          <EuiButtonEmpty
+            href="app/home#/tutorial/uptimeMonitors"
```

Comment:
```suggestion
            href="/app/home#/tutorial/uptimeMonitors"
```

Reply:
@justinkambic Thanks for the catch - I believe this is fixed in https://github.com/elastic/kibana/pull/69016/commits/cfecae6687098e024682973a5b12bfdcf82b1024#diff-d8ac796cb07610d1e13c47b996bfcbecR71

Reply:
@formgeist this is still sending me to a bad url like `/app/app/home#/tutorial/uptimeMonitors`.

I think if remove the `app` from the `getUrlForApp` call, we will end up with the correct URL.

```
href={kibana.services?.application?.getUrlForApp('home#/tutorial/uptimeMonitors')}
```